gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-client/src/main/java/org/apache/gobblin/service/FlowConfigClient.java

```diff
@@ -121,6 +123,25 @@ public void updateFlowConfig(FlowConfig flowConfig)
     response.getResponse();
   }
 
+  /**
+   * Partially update a flow configuration
+   * @param flowId flow ID to update
+   * @param flowConfigPatch {@link PatchRequest} containing changes to the flowConfig
+   * @throws RemoteInvocationException
+   */
+  public void partialUpdateFlowConfig(FlowId flowId, PatchRequest<FlowConfig> flowConfigPatch) throws RemoteInvocationException {
```

Comment:
Why is the partialUpdateFlowConfig() method duplicated in 3 different classes? Maybe move this to a Utils class?